processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/TableLoaderGenerator.java

```diff
@@ -69,6 +73,34 @@ private static String gtfsTypeToParserMethod(FieldTypeEnum typeEnum) {
         return "as" + CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, typeEnum.toString());
     }
 
+    private static boolean cachingEnabled(final GtfsFieldDescriptor field) {
```

Comment:
There may be a use case for that and we may have a command-line option to forcefully disable all caching. I added a FIXME since that would require to pass extra information to the table loader classes.

Anyway, this is not critical since you will see the actual performance difference only on large datasets and large datasets in the real-world would have a lot of cache hits.